packages/kit/src/core/postbuild/analyse.js

```diff
@@ -93,10 +93,11 @@ async function analyse({ manifest_path, env }) {
 				prerender = mod.prerender;
 			}
 
-			Object.values(mod).forEach((/** @type {import('types').HttpMethod} */ method) => {
-				if (mod[method] && ENDPOINT_METHODS.has(method)) {
-					api_methods.push(method);
-				} else if (mod.fallback) {
+			Object.keys(mod).forEach((key) => {
+				// @ts-ignore
+				if (mod[key] && ENDPOINT_METHODS.has(key)) {
+					api_methods.push(/** @type {import('types').HttpMethod} */ (key));
+				} else if (mod.fallback && key === 'fallback') {
 					api_methods.push('*');
 				}
```

Comment:
I think you can fix the ignore with something like:
```suggestion
				if (ENDPOINT_METHODS.has(key)) {
					const method = /** @type {import('types').HttpMethod} */ (key)
					if (mod[method]) {
						api_methods.push(method);
					} else if (mod.fallback) {
						api_methods.push('*');
					}
				}
```

Reply:
Thanks! I've added the eslint fix.